src/main/java/core/service/oparation/OperationStrategy.java

```diff
@@ -0,0 +1,5 @@
+package core.service.oparation;
+
+public interface OperationStrategy {
+    OperationHandler get(String typeOperations);
```

Comment:
у нас же є єнам TypeOperations
```suggestion
    OperationHandler get(TypeOperations typeOperations);
```

Reply:
I redid emthod get(), it has only one line now.